help/sources/tutorials/ui/create/advertising/pinterest-ads.md

```diff
@@ -0,0 +1,264 @@
+
+### Create Platform Schema {#create-platform-schema}
+
+Create a Platform [schema](/help/xdm/schema/composition.md) required for Linked In Ads.
+![Platform UI screenshot showing an example Schema](../../../../images/tutorials/create/advertising/pinterest-ads/schema.png?lang=en)
+
+You can obtain sample definitions [here](#schema-example).
+
+## Connect your [!DNL Pinterest Ads] account {#connect-account}
+
+In the Platform UI, select **[!UICONTROL Sources]** from the left navigation bar to access the [!UICONTROL Sources] workspace. The [!UICONTROL Catalog] screen displays a variety of sources with which you can create an account.
+
+You can select the appropriate category from the catalog on the left-hand side of your screen. Alternatively, you can find the specific source you wish to work with using the search option.
+
+Under the *Advertising* category, select *[!DNL Pinterest Ads]*, and then select **[!UICONTROL Add data]**.
+
+![Platform UI screenshot for catalog with Pinterest Ads card](../../../../images/tutorials/create/advertising/pinterest-ads/catalog.png?lang=en)
+
+The **[!UICONTROL Connect Pinterest Ads account]** page appears. On this page, you can either use new credentials or existing credentials.
+
+### Existing account {#existing-account}
+
+To use an existing account, select the [!DNL Pinterest Ads] account you want to create a new dataflow with, then select **[!UICONTROL Next]** to proceed.
+
+![Platform UI screenshot for Connect Pinterest Ads account with an existing account](../../../../images/tutorials/create/advertising/pinterest-ads/existing.png?lang=en)
+
+### New account {#new-account}
+
+If you are creating a new account, select **[!UICONTROL New account]**, and then provide a name, an optional description, and your credentials. Refer to the [prerequisites](#prerequisites) section for the values needed. When finished, select **[!UICONTROL Connect to source]** and then allow some time for the new connection to establish.
+
+![Platform UI screenshot for Connect Pinterest Ads account with a new account](../../../../images/tutorials/create/advertising/pinterest-ads/new.png?lang=en)
+
+If the details provided are valid, the UI displays a **[!UICONTROL Connected]** status with a green check mark, you can then proceed to the next step.
+
+![Platform UI screenshot for Connect Pinterest Ads account with connected status](../../../../images/tutorials/create/advertising/pinterest-ads/connected.png?lang=en)
+
+## Next steps {#next-steps}
+
+By following this tutorial, you have established a connection to your [!DNL Pinterest Ads] account. You can now continue on to the next tutorial and [configure a dataflow to bring data into Platform](/help/sources/ui-tutorials/dataflow/advertising.md).
+
+## Select data {#select-data}
```

Comment:
Let's move this select data step to before the next steps, then we can remove "dataflow detail", "mapping", and "review" since those steps are the modularized in the advertising dataflow tutorial